### ISSUE TITLE
Update Bundle to Port Later SensioGeneratorBundle Updates

### ISSUE THREAD

**Issue by ReservedDeveloper** [iambrosi/IsmaAmbrosiGeneratorBundle#11]
This may be a major rewrite of this bundle, but in looking through this bundle, I notice its missing a lot of the functionality (ex: template overrides) available in its pseudo-parent project, https://github.com/sensio/SensioGeneratorBundle. I believe this is because this bundle was created based on a *much* earlier version of Sensio's. 

It'd be great to see a v3 release with this. I've forked this repo and hope to start work on that soon, depending on how things go.

**Comment by iambrosi** [comment]
@ReservedDeveloper That would be awesome! I've been planning to do this myself, but I haven't had the time.

I'd like to help you, so feel free to ping me.

**Comment by ReservedDeveloper** [comment]
Definitely. Admittedly, I'm hoping for https://github.com/sensiolabs/SensioGeneratorBundle/pull/377 to be merged soon. I may wait on that. And, similar to you, also have some other time-demanding obligations to attend to. 

That said, knowing that you're also thinking about it will hopefully help keep me accountable :laughing:

**Comment by JHGitty** [comment]
You are right! I would love to have easier development in the future with SensioGeneratorBundle. Currently I cannot use it because it does not have Best Practices (PR is not merged yet) and some functionality are missing.
